crates/snforge-scarb-plugin/tests/integration/single_attributes/available_gas.rs

```diff
@@ -4,56 +4,99 @@ use indoc::formatdoc;
 use snforge_scarb_plugin::attributes::available_gas::available_gas;
 
 #[test]
-fn fails_with_empty() {
+fn works_with_empty() {
```

Comment:
Do we want to allow this? I'm not sure what was the use case of empty available gas previously.

Reply:
Maybe we could at least deprecate that?

Reply:
It works as if the attribute was not set basically, as then available_gas is set to max

Reply:
For me it's weird to support this, but if we also did support it previously then probably doesn't matter.